frontend/packages/kubevirt-plugin/integration-tests-cypress/view/dialog.ts

```diff
@@ -52,5 +55,49 @@ export const addDisk = (disk: Disk) => {
       .select(Cypress.env('STORAGE_CLASS'))
       .should('have.value', Cypress.env('STORAGE_CLASS'));
   }
+  if (disk.source === DISK_SOURCE.Url) {
+    if (disk.url) {
+      cy.get(diskDialog.sourceURL).type(disk.url);
+    } else {
+      throw new Error('No value for `disk.url` provided!!!');
+    }
+  } else if (disk.source === DISK_SOURCE.Container) {
+    if (disk.url) {
+      cy.get(diskDialog.container).type(disk.url);
+    } else {
+      throw new Error('No value for `disk.url` provided!!!');
+    }
+  } else if (
+    disk.source === DISK_SOURCE.AttachDisk ||
+    disk.source === DISK_SOURCE.AttachClonedDisk
+  ) {
+    if (disk.pvc) {
+      cy.get(diskDialog.diskPVC)
+        .select(disk.pvc)
+        .should('have.value', disk.pvc);
+    } else {
+      throw new Error('No value for `disk.pvc` provided!!!');
+    }
+  }
+  if (disk.autoDetach === true) {
+    cy.get(diskDialog.autoDetach)
+      .check()
+      .should('be.checked');
+  } else if (disk.autoDetach === false) {
+    cy.get(diskDialog.autoDetach)
+      .uncheck()
+      .should('not.be.checked');
+  }
   cy.get(diskDialog.add).click();
 };
+
+export const delDisk = (name: string) => {
+  cy.get(`[data-id="${name}"] ${kebabBtn}`).click();
+  cy.get(deleteDiskBtn).click();
+  cy.get(modalConfirmBtn).click();
+};
+
+export const waitForCurrentVMStatus = (status: string, timeout?: number) => {
```

Comment:
The name shouldn't be VMStatus, maybe diskStatus?

Reply:
I think the function name is a bit confusing, maybe should be changed from  `waitForCurrentVMStatus` to `waitForVMStatusLabel`, and moving it the `views/vm` is a good idea

Reply:
Agree, better to change the name and move it to views/vm.

Reply:
Moved to `views/vm` as `waitForVMStatusLabel`.